pkg/subctl/benchmark/latency.go

```diff
@@ -0,0 +1,56 @@
+package benchmark
+
+import (
+	"fmt"
+
+	"github.com/onsi/gomega"
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func StartLatencyTests() {
+	gomega.RegisterFailHandler(func(message string, callerSkip ...int) { panic(message) })
+	f := initFramework("latency")
+	framework.By("Performing latency tests from Gateway pod to Gateway pod")
+	runLatencyTest(f, framework.GatewayNode)
+	framework.By("Performing latency tests from Non-Gateway pod to Non-Gateway pod")
+	runLatencyTest(f, framework.NonGatewayNode)
+	cleanupFramework(f)
```

Comment:
Can you add blank lines between logical sections to improve readability? I'd suggest one before L14, L16, and L18.